Building-ChakraCore.md

```diff
@@ -45,13 +45,11 @@ msbuild ... /p:RuntimeLib=static_library ...
 
 ## Linux ##
 
-ChakraCore on Linux is still under development. We are working on a first implementation of ChakraCore interpreter and runtime,
-no JIT, on Linux, targeting x64 Ubuntu 16.04 LTS and Clang 3.8+.
-We will add JIT later as we progress. You can test the current support with the following instructions.
```

Comment:
We already have it on https://github.com/Microsoft/ChakraCore/wiki/Release-Notes ?